wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/WorkflowOutputDeclarationElement.scala

```diff
@@ -0,0 +1,16 @@
+package wdl.model.draft3.elements
+
+import common.validation.ErrorOr.ErrorOr
+import simulacrum.typeclass
+
+case class WorkflowOutputDeclarationElement(name: String,
+                                            valueType: String,
+                                            value: String
+                                ) extends LanguageElement {
+  override def children: Seq[String] = Seq(name, valueType, value)
```

Comment:
You're trying to override the `LanguageElement.children` field which is a `Seq[LanguageElement]` with a `Seq[String]` so this is an error -

I think in this case the declaration has no child `LanguageElements` (since the fields are all terminals) so I think this is just `Seq.empty`